rest/stat.go

```diff
@@ -1,15 +1,61 @@
 package rest
 
-// // GetQPSStats returns current queries per second (QPS) for the account
-// func (c APIClient) GetQPSStats() (v float64, err error) {
-// 	var s map[string]float64
-// 	_, err = c.doHTTPUnmarshal("GET", "https://api.nsone.net/v1/stats/qps", nil, &s)
-// 	if err != nil {
-// 		return v, err
-// 	}
-// 	v, found := s["qps"]
-// 	if !found {
-// 		return v, errors.New("Could not find 'qps' key in returned data")
-// 	}
-// 	return v, nil
-// }
+import (
+	"errors"
+	"fmt"
+)
+
+const statsQpsEndpoint = "stats/qps"
+
+// StatsService handles 'stats/qps' endpoint.
+type StatsService service
+
+//Returns current queries per second (QPS) for the account.
+// The QPS number is lagged by approximately 30 seconds for statistics collection;
+// and the rate is computed over the preceding minute.
+func (s StatsService) GetQps() (float32, error) {
+	return s.getQps(statsQpsEndpoint)
+}
+
+// Returns current queries per second (QPS) for a specific zone.
+// The QPS number is lagged by approximately 30 seconds for statistics collection;
+// and the rate is computed over the preceding minute.
+func (s StatsService) GetZoneQps(zone string) (float32, error) {
+	path := fmt.Sprintf("%s/%s", statsQpsEndpoint, zone)
+	return s.getQps(path)
+}
+
+// Returns current queries per second (QPS) for a specific record.
+// The QPS number is lagged by approximately 30 seconds for statistics collection;
+// and the rate is computed over the preceding minute.
+func (s StatsService) GetRecordQps(zone, record, t string) (float32, error) {
+	path := fmt.Sprintf("%s/%s/%s/%s", statsQpsEndpoint, zone, record, t)
+	return s.getQps(path)
+}
+
+func (s StatsService) getQps(path string) (float32, error) {
```

Comment:
Other handler Get functions also return a pointer to the http.Response.  Is there a reason we wouldn't want to do that in Stats?

Reply:
Updated to return qps and *http.Response (similar to https://github.com/ns1/ns1-go/blob/03b09e404c10ef6d966b038f97364710db4f972f/rest/account_setting.go)